packages/fxa-auth-server/test/local/routes/validators.js

```diff
@@ -8,6 +8,25 @@ const { assert } = require('chai');
 
 const validators = require('../../../lib/routes/validators');
 
+const validProductMetadata = {
+  webIconURL: 'https://example.com/static/icon.svg',
+  upgradeCTA: 'hello <a href="http://example.org">world</a>',
+  downloadURL: 'https://example.com/download',
+  appStoreLink: 'https://example.com/appStoreRedirect',
+  playStoreLink: 'https://example.com/playStoreRedirect',
+  'product:privacyNoticeURL':
+    'https://accounts-static.cdn.mozilla.net/legal/privacy',
+  'product:privacyNoticeDownloadURL':
+    'https://accounts-static.cdn.mozilla.net/legal/privacy.pdf',
+  'product:termsOfServiceURL':
+    'https://accounts-static.cdn.mozilla.net/legal/terms',
+  'product:termsOfServiceDownloadURL':
+    'https://accounts-static.cdn.mozilla.net/legal/terms.pdf',
+  productSet: '123done',
+  productOrder: 2,
+  'capabilities:aFakeClientId12345': 'more,comma,separated,values',
+};
```

Comment:
nit: this looks identical to some of the changes above.  Can we just require one of the plans from the fixtures and use the metadata in that plan?